src/test/java/org/cactoos/text/ReplacedTextTest.java

```diff
@@ -0,0 +1,24 @@
+package org.cactoos.text;
+
+import org.cactoos.TextHasString;
+import org.hamcrest.MatcherAssert;
+import org.junit.Test;
+
+/**
+ * Test case for {@link ReplacedText}.
+ * @author Mehmet Yildirim (memoyil@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class ReplacedTextTest {
+
+    @Test
+    public void replaceText() {
```

Comment:
You are incorrect because both replace() and replaceAll() replaces all the occurrence found. However, there is difference in the type of arguments they accept.

replace() works on CharSequence, where as replaceAll(), accepts regex, so you can do a regular expression based pattern matching.